ui/raidboss/timeline.js

```diff
@@ -787,29 +787,76 @@ class TimelineUI {
   }
 }
 
+const CountdownState = {
+  none: 0,
+  active: 1,
+  wipe: 2,
+};
+
 class TimelineController {
   constructor(options, ui) {
     this.options = options;
     this.ui = ui;
     this.dataFiles = {};
     // data files not sent yet.
     this.timelines = null;
+
+    // Fix for #1453
+    // Countdown state
+    // 0 = No countdown
+    // 1 = Countdown active
+    // 2 = Wiped during countdown
```

Comment:
I think the values aren't important to comment anymore.

Reply:
Agreed. Changing that comment block to this instead:
```javascript
    // Fix for #1453
    // Countdown state is tracked and if the bug is encountered (countdown, wipe, engage)
    // then the engage line is never passed to the timeline
```